### PR TITLE
Wishlist 和 cart 的 api

### PR DESCRIPTION
**Pull request by codax0421**
# Summary

# Frontend Changes

# Backend Changes

# User-facing changes

# Check List

- [ ] I have added documentation where necessary

- [ ] I have tested this code

### CODE REVIEW

**Comment by codax0421** [comment]
@Bosh-Kuo  已更新img style

File: e_commerce_frontend/src/components/Layout/Card.js [hunk 1]
```diff
@@ -60,8 +64,9 @@ const CardProduct = ({ products }) => {
             cover={
               <img
                 style={{
-                  width: "100%",
-                  height: "100%",
+                  width: " 300px",
+                  height: "200px",
+                  objectFit: " cover",
```
Comment (Bosh-Kuo) [inline e_commerce_frontend/src/components/Layout/Card.js]: 幫我改成這樣～
style={{ width: " 300px",height: 300px",objectFit: "contain"}}

File: e_commerce_backend/app/serializers.py [hunk 2]
```diff
@@ -92,16 +93,12 @@ def create(self, validated_data):
 class WishlistSerializer(serializers.ModelSerializer):
     class Meta:
         model = Wishlist
```
Comment (Bosh-Kuo) [inline e_commerce_backend/app/serializers.py]:
```suggestion
class WishlistSerializer(serializers.ModelSerializer):
    productName = serializers.CharField(source="product.name", read_only=True)
    userName = serializers.CharField(source="user.username", read_only=True)
    class Meta:
        model = Wishlist
```

File: e_commerce_backend/app/serializers.py [hunk 3]
```diff
@@ -92,16 +93,12 @@ def create(self, validated_data):
...
-        fields = ["id", "user", "product"]
+        fields ="__all__"
 
 
 class CartSerializer(serializers.ModelSerializer):
     class Meta:
         model = Cart
```
Comment (Bosh-Kuo) [inline e_commerce_backend/app/serializers.py]:
```suggestion
class CartSerializer(serializers.ModelSerializer):
    productName = serializers.CharField(source="product.name", read_only=True)
    userName = serializers.CharField(source="user.username", read_only=True)
    class Meta:
        model = Cart
```

**Review by Bosh-Kuo — COMMENTED** [review]
Serializer 加一下 productName 跟 userName

**Pull request merged**